deps.edn

```diff
@@ -38,7 +38,9 @@
            :benchmark {:main-opts ["-m" "benchmark.core"]
                        :extra-paths ["benchmark/src"]
                        :extra-deps {clj-http/clj-http     {:mvn/version "3.11.0"}
-                                    org.clojure/tools.cli {:mvn/version "1.0.194"}}}
+                                    org.clojure/tools.cli {:mvn/version "1.0.194"}
+                                    incanter/incanter-core {:mvn/version "1.9.2"} 
```

Comment:
incanter should also be added to the `bench-dev` environment.

Reply:
Now added to dev